docs/using.adoc

```diff
@@ -198,7 +197,9 @@ vagrant ssh
 You are now at a shell prompt inside the Vagrant box. You can now
 execute commands and use the tools provided.
 
-You can use the
+=== Using orchestrators
+
+You can use
 https://github.com/projectatomic/adb-utils/blob/master/README.adoc[sccli]
 to manage the orchestration services inside of ADB. `sccli` makes it
 easy to start and stop orchestration providers like Kubernetes or
```

Comment:
Now, it can manage other operations like `restart` and `status`.

```
[vagrant@rhel-cdk ~]$ sccli -h
usage: sccli [-h] {kubernetes,openshift,docker} ...

CLI for managing services in ADB/CDK

optional arguments:
  -h, --help            show this help message and exit

subcommands:
  Manage services for openshift|docker|kubernetes

  {kubernetes,openshift,docker}
    kubernetes          start|restart|status|stop (default:start)
    openshift           start|restart|status|stop (default:start)
    docker              start|restart|status|stop (default:start)
```

You many want to change `easy to start and stop` to something else.